webapp/components/Comment/Comment.vue

```diff
@@ -54,6 +54,15 @@
         </button>
       </div>
       <ds-space margin-bottom="small" />
+      <base-button
+        :title="this.$t('post.comment.reply')"
+        icon="level-down"
+        @click.prevent="reply"
```

Comment:
Please put the events to the end of the list. 👇🏼

Reply:
@mattwr18 it's true, we haven't been following this guideline very strictly and I'm not sure that having things sorted alphabetically would do us much good – like it's handled in many other projects – but I'm with @Tirokk when he says "please put events at the end of the list" and would add "please put v-if and v-show and v-for in first position". Those are the two things I usually "fix" when I come across a different order...

Reply:
That’s why I sad:
> Please put the class *more to the beginning* of the list, see other examples.

Thanks for clarification @alina-beck ! I should have explicitly said that `v-if`, `v-else`, `v-else-if`, `v-show`, and `v-for` should be always at the beginning.

Reply:
sorry, my comments were in `Discord`, so @alina-beck was responding to my comments there. 
I like the idea of having the events at the bottom, and the conditionals at the top, for sure.
I just didn't really like to ask @ogerly to change the location of his `class`es. The reasons, as I stated, are
```
1)  we haven't agreed as a team that is what we want to do and 
2) sometimes, I feel like we can ask @ogerly to change things, we don't ask others to do, 
```